openmdao/core/tests/test_group.py

```diff
@@ -514,42 +514,6 @@ def compute(self, inputs, outputs):
                          "Group (<model>): src_indices has been defined in both "
                          "connect('indep.x', 'C1.x') and add_input('C1.x', ...).")
 
-    def test_incompatible_src_indices_error(self):
```

Comment:
I don't think you mean to take this test out.  This is different than the one we talked about.

Reply:
Fixed